app/views/publish/providers/schools/edit.html.erb

```diff
@@ -1,24 +1,29 @@
 <% page_title = @school_form.site.location_name %>
 <% content_for :page_title, title_with_error_prefix(page_title, @school_form.errors.any?) %>
-<%= content_for :before_content, render_breadcrumbs(:edit_site) %>
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(publish_provider_recruitment_cycle_school_path(@provider.provider_code, @site.recruitment_cycle.year, @site.id)) %>
+<% end %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+        <%= @site.location_name_was %>
+      </h1>
+
     <%= form_with(
       model: @school_form,
       url: publish_provider_recruitment_cycle_school_path(@school_form.provider_code, @school_form.recruitment_cycle_year, @school_form.site.id),
-      method: :patch
+      method: :patch,
+      class: "school-form"
     ) do |f| %>
 
       <%= f.govuk_error_summary %>
 
-      <h1 class="govuk-heading-l">
-        <%= page_title %>
-      </h1>
-
-      <p class="govuk-body">When you’ve added a school, you can assign it to a course in the ‘Basic details’ tab of the course. Candidates will see a list of schools on the course page on Find.</p>
       <%= render "form", f: %>
-      <%= f.govuk_submit "Save and publish" %>
+      <%= f.govuk_submit(@site.persisted? ? t("support.update_record") : t("support.continue")) %>
```

Comment:
Don't we have non support specific keys for these translations? Not sure we want to use them in provider specific views. If not we can simply create non support ones and re-reference the original values in the yaml. Eg:

```
key: &value some_value
other_key: *value
```

Reply:
We don't have provider specific translations for many of these instances, I have added a few to the en.yml, but I think it could do with some refactoring, if the translations are identical, then perhaps generic keys e.g. might be DRYer than anchors?

Reply:
We don't have provider specific translations for many of these instances, I have added a few to the en.yml, but I think it could do with some refactoring, if the translations are identical, then perhaps generic keys e.g ('schools.whatever') might be DRyer than anchors?

Reply:
Yeah good shout